src/Login/index.test.js

```diff
@@ -8,68 +8,106 @@ import Login from './index';
 import QUERY_USERS from '../graphql/queries';
 import { HOME_PATH, LOGIN_PATH } from '../paths';
 
-const mockedUsersResponse = {
-  request: { query: QUERY_USERS },
-  result: {
-    data: {
-      users: [
-        {
-          username: 'User',
-          password: 'password',
-          id: 1,
-        },
-      ],
+const mocks = [
+  {
+    request: { query: QUERY_USERS },
+    result: {
+      data: {
+        users: [
+          {
+            username: 'User',
+            password: 'password',
+            id: 1,
+          },
+        ],
+      },
     },
   },
-};
-
-const mocks = [
-  mockedUsersResponse,
-  mockedUsersResponse,
-  mockedUsersResponse,
 ];
 
-it('renders the login page', async () => {
-  render(
-    <MockedProvider mocks={mocks} addTypename={false}>
-      <AuthProvider>
-        <Router initialEntries={[LOGIN_PATH]}>
-          <Switch>
-            <Route
-              exact
-              path={HOME_PATH}
-              component={() => <div>Home</div>}
-            />
-            <Route path={LOGIN_PATH} component={Login} />
-          </Switch>
-        </Router>
-      </AuthProvider>
-    </MockedProvider>,
-  );
-  expect(screen.getByRole('form')).toBeInTheDocument();
-  expect(screen.getByText('Register Now')).toBeInTheDocument();
-  const [loginButton] = screen.getAllByRole('button');
-  expect(loginButton).toBeDisabled();
+describe('Login page', () => {
+  const setup = () => {
+    render(
+      <MockedProvider mocks={mocks} addTypename={false}>
+        <AuthProvider>
+          <Router initialEntries={[LOGIN_PATH]}>
+            <Switch>
+              <Route
+                exact
+                path={HOME_PATH}
+                component={() => <div>Home</div>}
+              />
+              <Route path={LOGIN_PATH} component={Login} />
+            </Switch>
+          </Router>
+        </AuthProvider>
+      </MockedProvider>,
+    );
+    const [loginButton] = screen.getAllByRole('button');
+    const usernameField = screen.getByLabelText('Username');
+    const passwordField = screen.getByLabelText('Password');
+    return { loginButton, usernameField, passwordField };
+  };
+
+  it('renders the login page', () => {
+    const { loginButton, usernameField, passwordField } = setup();
+    expect(screen.getByRole('form')).toBeInTheDocument();
+    expect(screen.getByText('Register Now')).toBeInTheDocument();
+    expect(loginButton).toBeInTheDocument();
+    expect(usernameField).toBeInTheDocument();
+    expect(passwordField).toBeInTheDocument();
```

Comment:
These assertions are redundant since we already do the same thing inside `setup()` with `getBy*`:
```
    expect(loginButton).toBeInTheDocument();
    expect(usernameField).toBeInTheDocument();
    expect(passwordField).toBeInTheDocument();
```